include/ignition/sensors/CameraSensor.hh

```diff
@@ -143,6 +143,16 @@ namespace ignition
       /// \todo(iche033) Make this function virtual on Garden
       public: bool HasConnections() const;
 
+      /// \brief Check if there are any image subscribers
+      /// \return True if there are image subscribers, false otherwise
+      /// \todo(iche033) Make this function virtual on Garden
```

Comment:
Can you change `Garden` to `Harmonic` for the new functions added in this PR? 

Harmonic is now the unstable / unreleased version which we can break API/ABI.

I'll then take care of these todos when merging forward.

Reply:
will do!